upload_cleanup.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/phrase/phraseapp-go/phraseapp"
+)
+
+type UploadCleanupCommand struct {
+	*phraseapp.Config
+	ID             string `cli:"arg required"`
+	SuppressPrompt bool   `cli:"opt --yes desc='Don’t ask for confirmation'"`
+}
+
+func (cmd *UploadCleanupCommand) Run() error {
+
+	client, err := newClient(cmd.Config.Credentials)
+	if err != nil {
+		return err
+	}
+
+	return UploadCleanup(client, cmd)
+}
+
+func UploadCleanup(client *phraseapp.Client, cmd *UploadCleanupCommand) error {
+	q := "unmentioned_in_upload:" + cmd.ID
+	params := &phraseapp.KeysListParams{Q: &q}
+
+	var err error
+	page := 1
+
+	keys, err := client.KeysList(cmd.Config.DefaultProjectID, page, 25, params)
+	if err != nil {
+		return err
+	}
+
+	if len(keys) == 0 {
+		fmt.Println("There were no keys unmentioned in that upload.")
+		return nil
+	}
+
+	for len(keys) != 0 {
+		ids := make([]string, len(keys), len(keys))
```

Comment:
my (personal) taste is not not use make (in that scale the performance is not critical at all) but just `ids := []string{}` or (when using make) to use `make([]string, 0, len(keys)` and just use `append` then.